nucypher/characters/lawful.py

```diff
@@ -604,7 +624,7 @@ def _try_orient(self, treasure_map, alice_verifying_key):
             raise  # TODO: Maybe do something here?  NRN
 
     def get_treasure_map(self, alice_verifying_key, label):
-        map_identifier = self.construct_map_id(verifying_key=alice_verifying_key, label=label)
+        map_identifier = self.construct_map_id(relayer_verifying_key=alice_verifying_key, label=label)
```

Comment:
Nice turn-of-phrase.

Reply:
👍🏻 Renamed to "publisher" for accuracy and readability.  Meant to indicate "Policy and Treasure Map Publisher".